LevelMeter/LevelMeterSource.h

```diff
@@ -99,8 +99,8 @@ class LevelMeterSource
         float getAvgRMS () const
         {
             if (rmsHistory.size() > 0)
-                return std::sqrtf (std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0f) / rmsHistory.size());
-
+                return std::sqrt(std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0f) / static_cast<float>(rmsHistory.size()));
```

Comment:
Good catch! I have no clue how I ended up using sqrtf...
The second operand is implicitly converted to the same type like the result of std::accumulate. It didn't raise a warning here, but it's probably nicer to explicitly cast it.

Reply:
Yes, the explicit cast was just a quick insurance because I did not look up what accumulate actually does